src/components/verification-grid/verification-grid.ts

```diff
@@ -304,6 +326,15 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
       return;
     }
 
+    if (event.ctrlKey && event.key === "d") {
+      event.preventDefault();
+      this.removeSubSelection();
+
+      if (!this.isViewingHistory()) {
+        this.removeDecisionButtonHighlight();
+      }
```

Comment:
this leaves me with so many questions - but knowing we're planning on changing this i'm just going to let this slide

Reply:
Good catch. I no longer need this (and it may have even caused a bug)

This was originally in here so that when you pressed escape to cancel a decision, the decision button highlight was removed.

This made sense when we could only apply one decision per tile, and we only showed touched decision buttons when viewing history.